README.md

```diff
@@ -382,6 +382,26 @@ Or any other attributes for that matter:
 
 > ⚠️ Note that with Blade components, using a prefix is always required, even when referencing icons from the default set.
 
+#### Deferring icons
+
+When you're using the same icon in lots of places on the page the DOM element count may explode upwards.
+To remedy this you can add the defer attribute to the components:
+
+```blade
+<x-icon-camera :defer="true"/>
```

Comment:
Can't we use the below?
```suggestion
<x-icon-camera defer />
```

Reply:
You absolutely can! and i've updated the readme since it is more obvious to use it that way